src/main/java/net/neoforged/neoforge/common/extensions/IBlockExtension.java

```diff
@@ -219,7 +219,9 @@ default public boolean canHarvestBlock(BlockState state, BlockGetter level, Bloc
      * @param willHarvest The result of {@link #canHarvestBlock}, if called on the server by a non-creative player, otherwise always false.
      * @param fluid       The current fluid state at current position
      * @return True if the block is actually destroyed.
+     * @deprecated Use the tool stack sensitive version bellow.
```

Comment:
It's customary to `{@link #... the tool stack sensitive}` to the new method, so please add a link here and in the other file too. Also, you have two `l` in below.